src/main/java/spoon/support/reflect/code/CtJavaDocImpl.java

```diff
@@ -96,6 +99,69 @@ public String getShortDescription() {
 		}
 	}
 
+	@Override
+	public <E extends CtComment> E setContent(String content) {
+		// in Javadoc we parse the tags
+		parseTags(cleanComment(content));
```

Comment:
parseTags adds tags internally. Sp we should clear old tags here. Or did I overlook something?

Reply:
you're right! fixed in parseTags!